e2e/e2eutils/setup.go

```diff
@@ -127,6 +127,10 @@ func MakeDeployParams(t require.TestingT, tp *TestParams) *DeployParams {
 		ProposalThresholdTokens:         1,
 		VotesQuorumFractionPercent:      51,
 		ProposalExecutingLatencySeconds: 1,
+
+		ZKVerifierHashScalar: (*hexutil.Big)(hexutil.MustDecodeBig("0x201bf8cdf8299a6ab7711b7ed71fb7ee9448728d1c41caa1577e4f8dd6a0f33a")),
+		ZKVerifierM56Px:      (*hexutil.Big)(hexutil.MustDecodeBig("0xa3500fa181d574a461035b8ae73a29e1aca62ea606eb8e4847dd74760d2c177")),
+		ZKVerifierM56Py:      (*hexutil.Big)(hexutil.MustDecodeBig("0x3cab33eacc5d51c399712707c5df1500c93ad67be0a3a45bebe9d96119ac469")),
 	}
 
 	// Configure the DeployConfig with the expected developer L1
```

Comment:
Don't we need to add `ZKVerifierProxy` to `DeploymentsL1` in this file?

Reply:
Yes. We don't need it because we don't use ZKVerifier contract in e2e tests.